roxie/ccd/ccdserver.cpp

```diff
@@ -380,7 +380,8 @@ class IndirectSlaveContext : implements IRoxieSlaveContext, public CInterface
 static const StatisticsMapping actStatistics({StWhenFirstRow, StTimeElapsed, StTimeLocalExecute, StTimeTotalExecute, StSizeMaxRowSize,
                                               StNumRowsProcessed, StNumSlaves, StNumStarts, StNumStops, StNumStrands,
                                               StNumScansPerRow, StNumAllocations, StNumAllocationScans,
-                                              StTimeFirstExecute, StCycleLocalExecuteCycles, StCycleTotalExecuteCycles});
+                                              StTimeFirstExecute, StCycleLocalExecuteCycles, StCycleTotalExecuteCycles,
+                                              StTimeBlocked, StCycleBlockedCycles});
```

Comment:
This is new I think (didn't see in previous PR).
Not so applicable to Roxie..
Are we sure we want the albeit minor cost, of tracking what little blocked time there is in Roxie.
Perhaps should leave to a separate JIRA as none being tracked yet anyway, and we can discuss there.

Reply:
Roxie activities are calling ActivityTimeAccumulator::addStatistics().  As ::addStatistics() has been modified to update the blocked cycles, actStatistics has to be updated to include the StTimeBlocked (otherwise it fails with: assert(index < mapping.numStatistics()) failed - file: jstats.h, line 491).

Reply:
Richard, what do you think?  Are we in danger of storing lots of 0 size blocked times.  Should ActivityTimer only add a blocked time if it is non-zero?

Reply:
Does StCycleBlockedCycles need to be included in this list?  It is never set, but code in jlib may assume it is there.  That should possibly be changed to avoid the need to include it.

Reply:
I think we should avoid adding blocked time here at all, until we have an instance that needs it.